### PR TITLE
Add data insertion tab

### LINKED ISSUE #21: Add Insert Entries Page
# Summary
Add an "insert entry" tab to allow for the insertion of Entries into the database. **Just implementing skeleton functionality-- details to be figured out later**

### PR DESCRIPTION
**Pull request by Shawk031**
# Inserts data insertion tab
Closes #21 This is a really simple data insertion thing for the moment, mostly to shorten future PRs. Good to just lay out some groundwork.

## Nav Bar Changes
So I changed up the CSS for the Nav Bar a decent amount. This is partly because the Div tags were messing me up for a while, so I simplified things a little. Of course the "Browse Entries" and "Search Entries" items are no longer grouped on the left as they were previously, but I think the "justify-between" makes it look ok now that we have 4 times. Since this is a more important component let me know if there are any design things that you'd want to change

## **Important Notes**
I'm sending in another pull request after this one, JUST for the `insertEntry.tsx` file. This will improve the CSS for that file and make the form look more like the rest of the website (Specifically the `Search Entries` Tab). The CSS definitely has a ways to go

### CODE REVIEW

File: app/models/entry.server.ts [hunk 1]
```diff
@@ -50,6 +49,46 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export async function putEntries(data: any) {
```
Comment (hangler) [inline app/models/entry.server.ts:52]: Let's avoid naming it `putEntries` (unless there's a convention I'm missing!) when it creates an entry. `put` is typically associated with the REST `PUT` verb, which refers to an update of an existing entry. Something like `create` would probably be better.

Also, can this code actually create multiple entries, or does it just create one at a time?

**Review by hangler — COMMENTED** [review]
Good proof of concept for an entry creation page.

File: app/models/entry.server.ts (hunk 1 above)
Comment (Shawk031) [reply app/models/entry.server.ts]: yeah good point. I changed it to "insertEntry." Though "create" actually makes a lot of sense in hindsight

**Review by hangler — APPROVED** [review]
✅

**Pull request merged**